ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java

```diff
@@ -1185,29 +1184,29 @@ public int getMonitorGraphInterval(long testId, String targetIP, int imageWidth)
 					skipCount = 0;
 					String[] datalist = StringUtils.split(line, ",");
 					if ("null".equals(datalist[4]) || "undefined".equals(datalist[4])) {
```

Comment:
use `NULL_STRING` or "null" 
I think it's good to unite in one way.

Reply:
I didn't find out there's a code uses the string value "null". I'm gonna change it to use string constants